### PR TITLE
Add speaker edit view (fix #932)

### LINKED ISSUE #932: Request: Edit the name of the speaker while updating the CFP
I have created my account with a google login which have a very special first name and last name : 'Marjôlen Rusée'.
When I submitted the CFP, the speaker name that I provided, 'LAURENS Jérôme' has been somehow modified by adding the first name of the google account name into 'LAURENS Jérôme Rusée', which is not appropriate.
May we have the possibility to edit this name while updating the CFP for example?

### PR DESCRIPTION
**Pull request by umgelurgel**
Fixes #932

### CODE REVIEW

File: conference/cfp.py [hunk 1]
```diff
@@ -164,6 +164,44 @@ def update_proposal(request, talk_uuid):
...
+    conf = Conference.objects.current()
+    if not conf.cfp():
+        return HttpResponseForbidden()
+
+    speaker_form = AddSpeakerToTalkForm(
+        initial=extract_initial_speaker_data_from_user(request.user)
+    )
+
+    if request.method == 'POST':
+        speaker_form = AddSpeakerToTalkForm(request.POST)
```
Comment (artcz) [inline conference/cfp.py]: I think in this case we might want to update the form name (or created an alias for this form with some other name that better suits this view)

File: templates/ep19/bs/cfp/update_speakers.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{% extends "ep19/bs/base.html" %}
+
+{% load crispy_forms_tags static %}
+
+{% block morecss %}
+<link href="{% static "taggit_labels/css/taggit_labels.css" %}" type="text/css" media="all" rel="stylesheet" />
+{% endblock %}
+
+{% block morejs %}
+<script src="{% static "/js/taggit_labels_customised_with_max_number_of_tags.js" %}"></script>
```
Comment (artcz) [inline templates/ep19/bs/cfp/update_speakers.html]: This is not needed for speakers, same for the CSS above.

File: tests/test_new_cfp.py [hunk 3]
```diff
@@ -32,7 +35,6 @@ def test_if_cfp_pages_are_login_required(client, url):
     assert redirects_to(response, "/accounts/login/")
 
 
-@mark.django_db
```
Comment (artcz) [inline tests/test_new_cfp.py:35]: :+1:  
what we could also do is make `user_client` fixture explicitly depend on `db` fixture which should do the same in a slightly more explicit way.

Comment (umgelurgel) [reply tests/test_new_cfp.py:35]: Would that mean that all `django_db` tests would have to accept `user_client` as an argument?

File: templates/ep19/bs/cfp/update_speakers.html (hunk 2 above)
Comment (umgelurgel) [reply templates/ep19/bs/cfp/update_speakers.html]: Thanks for catching this, removing!

**Review by artcz — APPROVED** [review]
LGTM

File: tests/test_new_cfp.py (hunk 3 above)
Comment (artcz) [reply tests/test_new_cfp.py:35]: other way around – user_client would take db as argument, which it needs anyway because it creates users.

**Pull request merged**